packages/network/src/workers/Recover.worker.ts

```diff
@@ -1,5 +1,5 @@
 import { Message } from "@latticexyz/services/protobuf/ts/ecs-relay/ecs-relay";
-import { expose } from "threads";
+import { expose } from "threads/worker";
```

Comment:
gpt-4 figured this out after a couple rounds of pasting in build errors and trying out its suggestions 🤯

Reply:
🤯